RELEASE-NOTES.txt

```diff
@@ -1,9 +1,15 @@
 Unreleased
 ---
-* [*] Small refinement to media upload errors, including centring and tweaking copy. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4597]
 
-## 1.71.3
+## 1.72.0
+---
+* [*] Add GIF badge for animated GIFs uploaded to Image blocks [https://github.com/WordPress/gutenberg/pull/38996]
+* [*] Small refinement to media upload errors, including centering and tweaking copy. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4597]
+* [*] Update gesture handler and reanimated libraries [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4615]
+* [*] Pass the ol start and reversed to native RichText too [https://github.com/WordPress/gutenberg/pull/39354]
```

Comment:
Thanks for adding the missing release note @jhnstn !

WDYT about the following phrasing for it (the one from the PR title is probably too technical for the app stores):
```suggestion
* [*] Fix issue with list's starting index and the order [https://github.com/WordPress/gutenberg/pull/39354]
```

Reply:
Sounds good, I'll make the same update everywhere else.